scaffolding-chef/lib/scaffolding.sh

```diff
@@ -91,7 +91,7 @@ cache_path "$pkg_svc_data_path/cache"
 node_path "$pkg_svc_data_path/nodes"
 role_path "$pkg_svc_data_path/roles"
 
-ssl_verify_mode :verify_none
+ssl_verify_mode {{cfg.ssl_verify_mode}}
```

Comment:
I see this merged, but this change has issues.
This scaffolding generates "$pkg_prefix/.chef/config.rb". So I'm expecting that run any command from "$pkg_prefix" will automatically uses `config.rb`. But this file is a template. Copying this config to `bootstrap-config.rb` also useless because I cannot use this config until habitat supervisor starts and renders configuration.

Reply:
@jsirex I don't understand your comment here. scaffolding-chef drops a run hook and runs the config from there, using a policyfile and chef-zero, referencing the config with the -c option. So it's safe to assume the Supervisor has started and the template is rendered. You should not be using scaffolding-chef as a general purpose method of running the chef-client, or as a method for dropping the chef-client on a box. For that you should use `chef/chef-client` directly or as a package dependency.

Reply:
Ok. What is purpose for `bootstrap-config.rb`?
https://github.com/habitat-sh/core-plans/pull/1888/files/c42e07cecb91433ea50785c591af59a98b5cf1b3#diff-661ba08fe17befce81a3bc9c3d089631R89 - template goes into "$pkg_prefix/.chef/config.rb"

I need chef to bootstrap machine/supervisor, but I need supervisor to bootstrap with chef...